src/Sentry/Infrastructure/DebugDiagnosticLogger.cs

```diff
@@ -10,6 +10,7 @@ namespace Sentry.Infrastructure
     /// <remarks>
     /// Logger available when compiled in Debug mode. It's useful when debugging apps running under IIS which have no output to Console logger.
     /// </remarks>
+    [Obsolete("Logger doesn't work outside of Sentry SDK. Please use TraceDiagnosticLogger instead")]
```

Comment:
Would it be fair to say...

```suggestion
    [Obsolete("This logger only works in debug mode. Please use TraceDiagnosticLogger instead.")]
```

Or does it not work even if consumed from a project built in debug?

Reply:
I believe the call to `Debug.Write` doesn't get compiled when we compile our SDk in Release mode so my guess is that it won't work. But curious if I'm just wrong here

Reply:
You are right. I checked and it gets completely omitted by the compiler.